src/audio/dai-zephyr.c

```diff
@@ -1054,7 +1054,12 @@ static int dai_comp_trigger_internal(struct comp_dev *dev, int cmd)
 		dai_trigger_op(dd->dai, cmd, dev->direction);
 #else
 		dai_trigger_op(dd->dai, cmd, dev->direction);
-		ret = dma_stop(dd->chan->dma->z_dev, dd->chan->index);
```

Comment:
the prefix in the commit title is wrong. This isn't in zephyr/dma, its actually audio:dai-zephyr/host-zephyr